docs/notebooks/pulse-efficient-transpilation.ipynb

```diff
@@ -455,16 +510,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now, we are ready to perform transpilation. As an example, we will transpile to run on the backend `ibm_auckland`. We define two functions: one to transpile into a basis gate set includes the `ecr` gate, and the other includes the `rzx` gate."
+    "Perform transpilation. This example is transpiled to run on the backend `ibm_auckland`. Define two functions: one to transpile into a basis gate set that includes the `ECR` gate, and the other includes the `RZX` gate."
```

Comment:
The backend chosen was `ibm_cusco` (see the output of cell 3); however, here, it says the example is run on `ibm_auckland.` Suggest we keep the references to the backend consistent.